packages/middleware/src/handlers/prepareLogger/index.ts

```diff
@@ -0,0 +1,24 @@
+import type { Response, Request, NextFunction } from "express";
+import { LoggerManager, wrapLogger } from "../../loggerManager";
+
+export function prepareLogger(loggerManager: LoggerManager) {
+  return function (req: Request, res: Response, next: NextFunction) {
+    if (!res.locals.alokai) {
+      res.locals.alokai = {};
+    }
+    const logger = loggerManager.get(req.params.integrationName);
```

Comment:
is it possible to have undefined params? can we make it safer? eg. req?.params?.integrationName ?? ''

it will not hurt to do

Reply:
But, does it hurt to write safe code? Did you ever had situation that you thought something is not possible and then it happened?

Reply:
Ok, let it be like that if someone every creates new route and tries to use it:
```ts
const logger = loggerManager.get(req?.params?.integrationName);
    if (!req?.params?.integrationName) {
      console.error("prepareLogger middleware used for unsupported route");
    }
```